src/App.js

```diff
@@ -1,12 +1,80 @@
 import React from 'react';
 import './App.css';
+import logo from "./assets/screenshot-logo.png"
+import FruitCounter from "./components/FruitCounter.js"
+import TextInputField from "./components/TextInputField.js"
+import RadioButton from "./components/RadioButton";
+import Button from "./components/Button";
+
+// console.assert()
 
 function App() {
-  return (
-    <>
-      <h1>Fruitmand bezorgservice</h1>
-    </>
-  );
+
+    let resetState = false
```

Comment:
Naast het feit dat het wel werkt vind ik het niet fijn. Als je state wilt bewerken die lager in de "React boom" staat, dan zul je hier dus je state ook moeten bijhouden. Dat betekent dat je dus hier ook de useState's moet bijhouden voor alle fruitsoorten.